src/api/zoops.ts

```diff
@@ -118,4 +118,25 @@ zoopsRouter.post("/:id/faves", requireUser, async (req: any, res, next): Promise
     }
 })
 
+  // DELETE /api/zoops/:id/faves
+  zoopsRouter.delete("/:id/faves", requireUser, async (req: any, res, next): Promise<void> => {
+    const userId = req.user?.id;
+
+    try {
+        const { id } = req.params;
+        const fave = await prisma.fave.findFirstOrThrow({
+            where: {
+                zoopId: Number(id), 
+                faverId: Number(userId)
+            }
+        })
```

Comment:
You could return a slightly more relevant error instead here
```suggestion
        const fave = await prisma.fave.findFirst({
            where: {
                zoopId: Number(id), 
                faverId: Number(userId)
            }
        })
        if (!fave) {
          next({name: "NotFound", message: "Could not find fave"})
        } else { 
          res.send({fave});
        }
```